android/src/main/java/com/stripeterminalreactnative/ktx/TerminalExtensions.kt

```diff
@@ -1,7 +1,6 @@
 package com.stripeterminalreactnative.ktx
 
 import com.stripe.stripeterminal.Terminal
-import com.stripe.stripeterminal.external.UsbConnectivity
```

Comment:
From 2.22.0 Changelog [0]:
`Update: @UsbConnectivity annotation is no longer required for using USB reader connection features.`

[0] https://github.com/stripe/stripe-terminal-android/blob/master/CHANGELOG.md#2220---2023-07-07